lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -137,7 +144,7 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
                 selections.forEach(node => (node['isExternal'] = true));
                 const selectionWithoutDuplication = this.removeExistingSelection(selections);
                 this.fixIncompatibilityFromPreviousAndNewForm(selectionWithoutDuplication);
-                if (this.field.value.length === 1) {
+                if(!this.lastSentToViewers) {
```

Comment:
Again, instead of looking at if there are any files attached, we check if the viewers are displaying anything